management-api-server/src/main/java/com/datastax/mgmtapi/resources/v2/TableOpsResources.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Please see the included license file for details.
+ */
+package com.datastax.mgmtapi.resources.v2;
+
+import com.datastax.mgmtapi.ManagementApplication;
+import com.datastax.mgmtapi.resources.common.BaseResources;
+import com.datastax.mgmtapi.resources.models.Table;
+import com.datastax.oss.driver.api.core.cql.ResultSet;
+import com.datastax.oss.driver.api.core.cql.Row;
+import com.google.common.collect.Lists;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.ExampleObject;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import java.util.List;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.http.HttpStatus;
+
+@Path("/api/v2/ops/tables")
+public class TableOpsResources extends BaseResources {
+
+  public TableOpsResources(ManagementApplication application) {
+    super(application);
+  }
+
+  @GET
+  @Produces({MediaType.TEXT_PLAIN, MediaType.APPLICATION_JSON})
+  @ApiResponse(
+      responseCode = "200",
+      description = "Table list",
+      content =
+          @Content(
+              mediaType = MediaType.APPLICATION_JSON,
+              array = @ArraySchema(schema = @Schema(implementation = Table.class))))
+  @ApiResponse(
+      responseCode = "400",
+      description = "Keyspace name not provided",
+      content =
+          @Content(
+              mediaType = MediaType.TEXT_PLAIN,
+              schema = @Schema(implementation = String.class),
+              examples =
+                  @ExampleObject(
+                      value = "List tables failed. Non-empty 'keyspaceName' must be provided")))
+  @Consumes(MediaType.APPLICATION_JSON)
+  @Operation(summary = "List the table names in the given keyspace", operationId = "listTables")
```

Comment:
Suggestion:
```suggestion
  @Operation(summary = "List the table names in the given keyspace", operationId = "listTablesV2")
```
So that the generated client creates a method called `listTablesV2` instead of leaving it to the client generation to avoid duplicate methods. Currently, the client generated will call this method `listTables1` which is not clear that it is actually a V2 implementation.